src/System.Data.SqlClient/src/System/Data/SqlClient/SNI/SNITcpHandle.cs

```diff
@@ -57,6 +57,12 @@ public override void Dispose()
                     _sslStream.Dispose();
                     _sslStream = null;
                 }
+
+                if (_tcpStream != null)
+                {
+                    _tcpStream.Dispose();
```

Comment:
Added tracking issue for EnableSsl/DisableSsl here: https://github.com/dotnet/corefx/issues/15669